package.json

```diff
@@ -11,19 +11,20 @@
   "bugs": "https://github.com/ain/smartbanner.js/issues",
   "dependencies": {},
   "devDependencies": {
+    "add-banner": "ain/add-banner#121b56d",
```

Comment:
nmp install does reorder alphabetically on new launch.
I can revert it if you want to :)
the only real change was that it was missing "browserify": "^14.3.0"